### PR TITLE
Remove VIP in cloud-to-prod interop tests.

### LINKED ISSUE #10763: cloud_to_prod and cloud_to_prod_auth interop tests are flaky against "default" and "gateway_v4".
Seen at https://grpc-testing.appspot.com/job/gRPC_interop_pull_requests/13464/ and https://grpc-testing.appspot.com/job/gRPC_interop_pull_requests/13477/.

They all seem to be reporting connection failures

### PR DESCRIPTION
**Pull request by adelez**
This fixes #10763.

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: tools/run_tests/run_interop_tests.py [hunk 1]
```diff
@@ -1477,19 +1465,14 @@ def aggregate_http2_results(stdout):
...
-        args.prod_servers, args.http2_interop)
-
     if num_failures:
         sys.exit(1)
     else:
         sys.exit(0)
 except Exception as e:
     print('exception occurred:')
     traceback.print_exc(file=sys.stdout)
+    sys.exit(1)
```
Comment (jtattermusch) [inline tools/run_tests/run_interop_tests.py]: Why do we want to throw here? Looks like this would affect the cleanup  that happens in the finally block?

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: tools/run_tests/run_interop_tests.py (hunk 1 above)
Comment (adelez) [reply tools/run_tests/run_interop_tests.py]: Kokoro will shows success even if there's an error.

Comment (jtattermusch) [reply tools/run_tests/run_interop_tests.py]: Yes, you're right.

**Comment by adelez** [comment]
Hmm, I don't understand why the interop tests keep failing, not just auth tests. Trying one more time.

File: tools/run_tests/run_interop_tests.py (hunk 1 above)
Comment (adelez) [reply tools/run_tests/run_interop_tests.py]: Got it. I'll do a separate PR.

**Comment by adelez** [comment]
I noticed only C-based languages are failing. @yang-g

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by adelez** [comment]
It seems that adding --server_host_override= flag will fix the test but it shouldn't be needed.

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Pull request merged**